packages/editor/styles.css

```
@@ -0,0 +1,62 @@
+.tuturuuu-editor-tool:hover > .tuturuuu-editor-tooltip,
+.tuturuuu-editor-tool:focus-within > .tuturuuu-editor-tooltip {
+  transform: translate(-50%, 0);
+  opacity: 1;
+}
+
+.tuturuuu-editor-toolbar-separator {
+  align-self: stretch;
+  width: 1px;
+  min-height: 1.5rem;
+  margin: 0 0.25rem;
+  background: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)